packages/swap/src/routes/swap.ts

```diff
@@ -268,7 +273,7 @@ router.post(
     const result = openSwapDepositChannelSchema.safeParse(req.body);
     if (!result.success) {
       logger.info('received bad request for new swap', { body: req.body });
-      throw ServiceError.badRequest('invalid request body');
+      throw ServiceError.badRequest('bad-request', 'invalid request body');
```

Comment:
Could we provide more info as to which fields were invalid?

Reply:
i was looking into generating a message for zod validation errors yesterday but decided to not include that in this pr bc it was not trivial. but i agree that it would be nice to give a hint about the validation error later on 🙂